skorch/utils.py

```diff
@@ -552,6 +554,58 @@ def check_is_fitted(estimator, attributes, msg=None, all_or_any=all):
         raise NotInitializedError(msg % {'name': type(estimator).__name__})
 
 
+def _identity(x):
+    """Return input as is, the identity operation"""
+    return x
+
+
+def _sigmoid_then_2d(x):
+    """Transform 1-dim logits to valid y_proba
+
+    Sigmoid is applied to x to transform it to probabilities. Then
+    concatenate the probabilities with 1 - these probabilities to
+    return a correctly formed ``y_proba``.
```

Comment:
Maybe specify who expects this format. Something along the lines of `formats outputs for use with BCE loss` or something like that